interview-questions.md

```diff
@@ -9,57 +9,77 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. Why would you use the method super()?
 
   Your answer:
+  super() is necessary in react when calling on classes.  It allows old constructors to be inherited.
```

Comment:
not just in react but in plain Javascript too. It's not necessarily that it inherits the constructor but rather inherits properties and behavior